.github/workflows/ci.yml

```diff
@@ -56,13 +56,20 @@ jobs:
         os: ["ubuntu-20.04", "ubuntu-22.04"]
```

Comment:
fix typo in commit message, `working` instead of `wotking`

Reply:
👍